src/GovUk.Education.ExploreEducationStatistics.Admin/Services/ReleaseService.cs

```diff
@@ -499,7 +509,9 @@ public async Task<Either<ActionResult, ReleaseViewModel>> UpdateReleaseVersion(
             Guid releaseVersionId, ReleaseUpdateRequest request)
         {
             return await ReleaseUpdateRequestValidator.Validate(request)
-                .OnSuccess(async () => await CheckReleaseVersionExists(releaseVersionId))
+                .OnSuccess(async () =>
+                    await _persistenceHelper.CheckEntityExists<ReleaseVersion>(releaseVersionId,
+                        queryable => queryable.Include(rv => rv.Release)))
                 .OnSuccess(_userService.CheckCanUpdateReleaseVersion)
                 .OnSuccessDo(async releaseVersion =>
                     await ValidateReleaseSlugUniqueToPublication(request.Slug,
```

Comment:
We've not planned to work on EES-5658 for a while until release labelling is added, but I imagine we'd want to do it earlier when we begin to make changes to this method for [EES-5626](https://dfedigital.atlassian.net/browse/EES-5626). As part of that change we'd update things to validate that it's unique including label I think?

If you think it would help to do it as part of this change I can do so. I've avoided any code changes which use the new relationship up until now so that it's not used until at least the migration has been deployed and tested. The only changes that are using the new relationship so far should be in unit tests.

Reply:
Yeah I agree, let's do it in EES-5626 :)